@asdffer can you fix Error: internal/processor/processor.go:81:60: slog.InfoContext arg "allowedMissingSegments" should be a string or a slog.Attr (possible missing key or value)
?   	github.com/javi11/nzb-touch	[no test files]
?   	github.com/javi11/nzb-touch/cmd/nzbtouch	[no test files]
?   	github.com/javi11/nzb-touch/internal/config	[no test files]
?   	github.com/javi11/nzb-touch/internal/nzb	[no test files]
FAIL	github.com/javi11/nzb-touch/internal/processor [build failed]
FAIL
make: *** [Makefile:47: test] Error 1

--- a/internal/processor/processor.go
+++ b/internal/processor/processor.go
@@ -78,7 +78,7 @@ func (p *Processor) ProcessNZB(ctx context.Context, nzb *nzbparser.Nzb, checkPer
 	// Calculate allowed missing segments based on TOTAL segments in NZB
 	allowedMissingSegments := (totalSegmentsInNZB * missingPercent) / 100
 
-	slog.InfoContext(ctx, "Total allowed missing segments: ", allowedMissingSegments)
+	slog.InfoContext(ctx, "Total allowed missing segments", "allowedMissingSegments", allowedMissingSegments)
 
 	// Track failed segments across entire NZB
 	var failedSegments int

